_posts/2021-05-19-gsoc-students.md

```diff
@@ -36,7 +36,7 @@ structure. For ionic systems, like batteries and many enzymes, the solvation
 environment surrounding ions is especially important. By studying the solvation
 of interesting materials, scientists can better understand, engineer, and
 design new technologies. The aim of this project is to implement a robust
-and cohesive set of methods for solvation analysis that would be widely
+and cohesive set of methods for [solvation analysis][orion-project] that would be widely
```

Comment:
I would give it a longer link, like Estefania's -- maybe [a robust and cohesive set of methods for solvation analysis] ?

Reply:
Note: cohesive set repeats in the sentence after, probably best to change it here if we can.